crates/uv-resolver/src/flat_index.rs

```diff
@@ -59,8 +58,58 @@ impl FlatIndex {
         Self { index, offline }
     }
 
+    /// Get the [`FlatDistributions`] for the given package name.
+    pub fn get(&self, package_name: &PackageName) -> Option<&FlatDistributions> {
+        self.index.get(package_name)
+    }
+
+    /// Returns `true` if there are any offline `--find-links` entries.
```

Comment:
This field docs say:

    /// Whether any `--find-links` entries could not be resolved due to a lack of network
    /// connectivity.

which sounds different from this docstring.